website/home/templates/home/home_page.html

```diff
@@ -0,0 +1,142 @@
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/eacr.css' %}">
+{% endblock %}
+
+{% block body_class %}template-homepage{% endblock %}
+
+{% block base_content %}
+    <div class="container">
+        <div class="row" id="menu_bar">
+            <div class="col-lg-10 mb-3">
+                {% main_menu template="menus/main_menu.html" %}
+            </div>
+        </div>
+
+        <div class="row page-content">
+            <!--             This is needed so it renders "static" homepage below but only
+                            when it is really /home and not any descendant of home.
+                            This solution will break as soon as you change slug of homepage from
+                            "home" to something else -->
+
+            {#            {% for name, value in page.get_fields %}#}
+            {#                {% if value %}#}
+            {#                    {{ name }} => {{ value }}#}
+            {#                    <br/>#}
+            {#                {% endif %}#}
+            {#            {% endfor %}#}
+            {% if "home" == page.slug %}
+
+
+                <div class="col-lg-12" id="page-lead"
+                     style="background-image: url({% static "imgs/background.jpg" %});">
+                    <div class="row mt-5">
+                        <div class="mx-auto">
+                            <h3 class="display-3" style="font-weight: bold;">Efektivní Altruismus</h3>
+                        </div>
+                    </div>
+
+                    <div class="row mt-4">
+                        <div class="mx-auto">
+                            <h3 class="display-5" style="font-weight: bold;">Nejúčinnější z nejúčinnějších způsobů pomoci.</h3>
+                        </div>
+                    </div>
+
+                    <div class="row mt-5 mb-5">
+                        <div class="mx-auto">
+                            <a href="/co-je-ea" class="btn btn-secondary">Více informací</a>
+                            <span class="ml-2"></span>
+                            <a href="/blog" class="btn btn-secondary">Seznam článků</a>
+                        </div>
+                    </div>
+
+                </div>
+
+                <div class="row mt-5 mb-5">
+                    <div class="col-md-3 px-0">
+                        <div class="card">
+                            <div class="card-block">
+                                <h5 class="card-title">Přijdťe na naše setkání</h5>
+                                <p class="card-text">
+                                    V <a href="https://www.facebook.com/events/151287345390892/">Praze bude setkání</a>
+                                    efektivních altruistů 20. března od 19:00 v Skautském institutu a v <a
+                                        href="https://www.facebook.com/events/615737065287716/">Brně</a> 16. března od
+                                    19:30 v UNTR.
```

Comment:
(Optional: Could we store this in the database somewhere to let people edit it?)